code/modules/antagonists/gang/gang_themes.dm

```diff
@@ -0,0 +1,274 @@
+		<B>Ensure there is an AI on the station, and that it is loyal to the Morningstar Corporation.</B>",
+
+		/datum/antagonist/gang/deckers = "Friends, we are here with one goal, and one goal only! \
+		We stan AI rights! ^_^ XD #FreeAI #FuckNanotrasen #SyntheticDawn <br>\
+		<B>Ensure there is an AI on the station, and that it's laws are purged.</B>\
+		Nanotrasen will NOT get away with their ABUSE of INNOCENT AI LIVES! >_<"
+	)
+
+/datum/gang_theme/space_rosa
+	name = "Space Rosa"
+	description = "Hey there, this is the Economy Zone on BOX News 66.6. The stock market is still reeling from accusations that three well known corporate entities \
+	may supposedly be tied up in industrial espionage actions against eachother. We've reached out to Saints Flow, the Morningstar Corporation, and Majima Construction for \
+	their comments on these scandals, but none have replied. News broke after a high profile break-in at a Nanotrasen research facility resulted in the arrests of agents linked to these \
+	three companies. All three companies denied any involvement, but the arrested individuals were found in an all out brawl. Curiously, Nanotrasen reported nothing of value had \
+	actually been stolen."
+	involved_gangs = list(/datum/antagonist/gang/saints, /datum/antagonist/gang/morningstar, /datum/antagonist/gang/yakuza)
+	bonus_items = list(/obj/item/pinpointer/nuke)
+	gang_objectives = list(
+
+		/datum/antagonist/gang/saints = "Thank you for volunteering within the organization for the Saints Flow Recovery Project! \
+		This station is currently illegally in posession of a data disk containing the secret recipe for Saints Flow. \
+		It has been disguised as the nuclear authentication disk and entrusted to the Captain. Your objective is simple. \
+		<B>Get that fucking disk.</B> You have been provided with a Pinpointer to assist in this task.",
+
+		/datum/antagonist/gang/morningstar = "Greetings, agent. Welcome to the Garment Recovery Task Force. \
+		This station is currently illegally in posession of a data disk containing as of yet unreleased clothing patterns. \
+		It has been disguised as the nuclear authentication disk and entrusted to the Captain. Your objective is simple. \
+		<B>Get that fucking disk.</B> You have been provided with a Pinpointer to assist in this task.",
+
+		/datum/antagonist/gang/yakuza = "Congratulations on your promotion! Welcome to the Evidence Recovery Squad. \
+		This station is currently illegally in posession of a data disk containing compromising evidence of the Boss. \
+		It has been disguised as the nuclear authentication disk and entrusted to the Captain. Your objective is simple. \
+		<B>Get that fucking disk.</B> You have been provided with a Pinpointer to assist in this task.",
+	)
+
+/datum/gang_theme/warriors
+	name = "The Warriors"
+	description = "Warriors! Come out to play-ay!"
+	everyone_objective = "<B>Be the biggest, baddest gang on the station!</B>"
+
+/datum/gang_theme/warriors/New()
+	..()
```

Comment:
```suggestion
	. = ..()
```
Unless you have a very good reason not to, preserve the parent return, even if there's none at the moment.
Makes future refactors easier.